### PR TITLE
add powerpoint mimetypes

### PR DESCRIPTION
**Pull request by valentinyanakiev**
## Summary by CodeRabbit

- **New Features**
  - Expanded support for PowerPoint document types with new MIME types added.
  
- **Database Updates**
  - Introduced a migration to update the `allowedMimeTypes` for PowerPoint documents in the storage system.

### CODE REVIEW

**Review by hero101 — APPROVED** [review]
looks ok, the added down migration by the rabbit might be useful

File: src/migrations/1734087106786-powerpointMimeTypes.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            ELSE CONCAT(allowedMimeTypes, ',${mimeType}')
+          END
+        WHERE FIND_IN_SET('${mimeType}', allowedMimeTypes) = 0;
+      `);
+    }
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    console.log('No down migration needed');
+  }
```
[the remark this answers is not in the document]
Comment (valentinyanakiev) [reply src/migrations/1734087106786-powerpointMimeTypes.ts:31]: I don't want to revert that migration, there is no point in going back.

File: src/migrations/1734087106786-powerpointMimeTypes.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      await queryRunner.query(`
+        UPDATE storage_bucket
+        SET allowedMimeTypes =
+          CASE
+            WHEN allowedMimeTypes IS NULL OR allowedMimeTypes = '' THEN '${mimeType}'
+            ELSE CONCAT(allowedMimeTypes, ',${mimeType}')
+          END
+        WHERE FIND_IN_SET('${mimeType}', allowedMimeTypes) = 0;
+      `);
+    }
```
[the remark this answers is not in the document]
Comment (valentinyanakiev) [reply src/migrations/1734087106786-powerpointMimeTypes.ts:26]: SQL injection is not possible in a migration.

**Comment by valentinyanakiev** [comment]
> looks ok, the added down migration by the rabbit might be useful

I don't believe so, there is no point in going back with this...

**Pull request merged**